adagrams/game.py

```diff
@@ -1,11 +1,112 @@
+import random
+from random import shuffle
+from collections import Counter
+
 def draw_letters():
-    pass
+    letters_pool = {
+        "A" : 9,
+        "B" : 2,
+        "C" : 2,
+        "D" : 4,
+        "E" : 12,
+        "F" : 2,
+        "G" : 3,
+        "H" : 2,
+        "I" : 9,
+        "J" : 1,
+        "K" : 1,
+        "L" : 4,
+        "M" : 2,
+        "N" : 6,
+        "O" : 8,
+        "P" : 2,
+        "Q" : 1,
+        "R" : 6,
+        "S" : 4,
+        "T" : 6,
+        "U" : 4,
+        "V" : 2,
+        "W" : 2,
+        "X" : 1,
+        "Y" : 2,
+        "Z" : 1
+    }
+    letters_probability = []
+    letter_bank = []
+    total = sum(letters_pool.values())
+    for letter in letters_pool:
+        p = letters_pool[letter] / total
+        letters_probability.append(p)
+    letters = list(letters_pool)
+
+    while len(letter_bank) < 10:
+        get_letter = random.choices(letters, weights=letters_probability, k=1)
```

Comment:
[nit] Would `random_letter`/`random_letters` be a more accurate name? If it doesn't feel too long/convoluted, I could imagine one-line:

```python
        random_letter = random.choices(letters, weights=letters_probability, k=1)[0]
```

which would replace all 3 instances of `get_letter[0]` below.